src/service/iot_config.proto

```diff
@@ -365,6 +365,60 @@ message admin_remove_key_req_v1 {
 
 message admin_key_res_v1 {}
 
+message gateway_info {
+  /// The asserted h3 (string) location of the gateway.
+  // Empty string if the gateway is not found
+  string location = 1;
+  bytes address = 2;
+  bytes owner = 3;
+  bool is_full_hotspot = 4;
+  /// the transmit gain value of the gateway in dbi x 10
+  // For example 1 dbi = 10, 15 dbi = 150
+  int32 gain = 5;
+  region region = 6;
+}
+
+/// Look up the details of a given hotspot public key
+message gateway_info_req_v1 {
+  /// The pubkey_bin address of the gateway to look up
+  bytes address = 1;
+  // sig from a key known to the config service
+  bytes signature = 2;
+}
+
+message gateway_info_res_v1 {
+  /// Timestamp of response
+  uint64 timestamp = 1;
+  gateway_info info = 2;
+  // sig from the config service
+  bytes signature = 3;
+}
+
+/// Request a stream of all active gateways
+message gateway_info_stream_req_v1 {
+  uint32 batch_size = 1;
+  // sig from a key known to the config service
+  bytes signature = 2;
+}
+
+/// Active gateway info streaming response containing a batch of gateways
+message gateway_info_stream_res_v1 {
+  repeated gateway_info_res_v1 gateways = 1;
+}
```

Comment:
```suggestion
message gateway_info_stream_res_v1 {
  repeated gateway_info gateways = 1;
  uint64 timestamp = 2;
  bytes signature = 3;
}
```
i know this seems redundant with the existing `gateway_info_res_v1` but since the info stream returns a stream of this message type as responses does it make sense to lift the metadata of the request up to the response-level message? when we validate the signature on these streaming responses we already need to grab the first message from the stream and verify the signature and with this structure we'd need to grab the first element of the batch from the first message in the stream to verify and it will probably be more performant to grab one timestamp and get one signature from the config service per batch as opposed to timestamping and signing for every one of the 500-700K gateways this api returns